lightning/src/ln/wire.rs

```diff
@@ -464,6 +482,19 @@ impl Encode for msgs::AcceptChannelV2 {
 	const TYPE: u16 = 65;
 }
 
```

Comment:
Let's go ahead and add the [`stfu`](https://github.com/lightning/bolts/blob/a68d7b07cea0fb243f25e955cca6134cae5872cf/02-peer-protocol.md#stfu) message so long for quiescence, what do you think?

Reply:
Agreed

Reply:
I have added `stfu` as well.